### ISSUE TITLE
Add support for Android 13 Notification permission

### ISSUE THREAD

**Issue by fabiendem** [zoontek/react-native-permissions#688]
### Why it is needed?

Android 13 will ship with a new permission: Notification permission.
https://developer.android.com/about/versions/13/changes/notification-permission

### Possible implementation

I suppose it could fit in the existing API `check/requestNotifications`

### Code sample

```js
Sorry I don't have a snippet for now
```

**Comment by mikehardy** [comment]
We're working on this already for Notifee and have a PR queued but are waiting to make sure Tiramisu / 13 stabilizes before landing. It's nothing fancy though. Just cross-pollinating in here in case it is useful at all

```java
public void requestPermission(Activity activity, MethodCallResult<Bundle> result) {
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
      requestPermissionCallResult = result;
      ActivityCompat.requestPermissions(activity, new String[]{Manifest.permission.POST_NOTIFICATIONS}, REQUEST_CODE_NOTIFICATION_PERMISSION);
    } else {
      getNotificationSettings(result);
    }
  }

  public boolean onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
    if (requestCode == REQUEST_CODE_NOTIFICATION_PERMISSION) {
      if (requestPermissionCallResult != null) {
        getNotificationSettings(requestPermissionCallResult);
        return true;
      }
    }
    return false;
  }
```

https://github.com/invertase/notifee/pull/351/files#diff-f5d7290ccfe968a3611168b4c271053d229241b15675d95bafdf35c32b3f5f23R424-R441

**Comment by zoontek** [comment]
This feature has landed in [v3.5.0](https://github.com/zoontek/react-native-permissions/releases/tag/3.5.0)

**Comment by fabiendem** [comment]
Merci!
